tests/test/application.js

```diff
@@ -43,9 +43,9 @@ module.exports = {
         ["home-constructionType", "Frame With Less Than 10% Brick"],
         ["home-roof-roofMaterial", "Metal"],
         ["home-roof-roofMaterialType", "Tin"],
-        ["home-roof-roofShape", "Complex/Custom"],
```

Comment:
I'm changing the value for this question to prevent YA partners from running into an issue I encountered. There's a weird spacing issue here. Don't know if it's a Mac/Linux to Windows or what, but the only way I could get this question to pass is by replacing the this text (`Complex/Custom`) with itself, as long as I typed it out.